acquire/tools/decrypter.py

```diff
@@ -444,6 +461,22 @@ def main():
                     executor.shutdown(wait=True, cancel_futures=True)
                     break
 
+            if len(files) > 1:
```

Comment:
Why only for more then 1 file?

Reply:
I thought it would be a bit overkill to show a summary when you're only targeting one file.

So, happy to show the results also for the one file case

Reply:
It would make the output more uniform ;-)